sansio_lsp_client/structs.py

```diff
@@ -83,10 +90,20 @@ def calculate_length(self, text: str) -> int:
 class TextDocumentContentChangeEvent(BaseModel):
     text: str
     range: t.Optional[Range]
-    rangeLength: t.Optional[int]
+    rangeLength: t.Optional[int]  # deprecated, use .range
```

Comment:
Maybe deprecated fields could be an interesting use for [typing.Annotated](https://devdocs.io/python~3.9/library/typing#typing.Annotated)?